.github/workflows/docs.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - run: python -m pip install --upgrade pip
       - run: python -m pip install -r requirements/ci.txt
       - run: |
-          sleep 30  # PyPI upload have a small delay until visible, avoid workflow fails in release builds
+          sleep 60  # PyPI upload have a small delay until visible, avoid workflow fails in release builds
```

Comment:
Can this be run in a loop, trying every 30s or so to download the package? Could use `pip download scipp`. Once that succeeds, we can call tox which can then install the package and doesn't even have to de-download it (hopefully).

Reply:
Let us see if this work, if it does not I will try your suggestion.

Reply:
@jl-wynen You were right, of course. Could you try to implement something along the lines of your suggestion? Thanks!